readme.md

```diff
@@ -54,3 +53,80 @@ Each Pull Request will be checked against following rules:
 - `commit/whitespace` - all changed files cannot contain TABs inside them. Before doing any work we suggest executing our `git_setup.cmd`. This will install git pre-commit hook that will normalize all whitespaces during commit
 - `commit/message/conventions` - all commit messages (except in merge commits) must contain issue number from our [YouTrack](http://issues.hibernatingrhinos.com) e.g. 'RavenDB-1234 Fixed issue with something'
 - `tests` - this executes `build.cmd Test` on our CI to check if no constraints were violated
+
+
+## Setup & Run
+
+Prerequsites:
+------------
+
+Windows
+-------
+Microsoft Visual C++ 2015 Redistributable Package should be installed prior to RavenDB launch.
+[Visual C++ Downloads](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads)
+See also: [Windows Prerequisites](https://docs.microsoft.com/en-us/dotnet/core/windows-prerequisites)
+
+
+Linux/MacOS
+-----------
+It is recommended that you update your OS before launching an instance of RavenDB.
+For example, Ubuntu-16.x as an updated OS doesn't require any additional packages.
+libsodium (1.0.13 or up) must be installed prior to RavenDB launch. 
+```
+In Ubuntu 16.x: apt-get install libsodium-18 
+In MacOS 10.12: brew install libsodium
+```
+You might need to also install additional packages, for example, 
+```
+apt-get install libunwind8 liblttng-ust0 libcurl3 libssl1.0.0 libuuid1 libkrb5 zlib1g libicu55
+```
+
+See also: [Linux Prerequisites](https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites) or 
+[MacOS Prerequisites](https://docs.microsoft.com/en-us/dotnet/core/macos-prerequisites)
+
+Lauch RavenDB:
+-------------
+Running locally:
+```
+<path/to/ravendb>/Server/Raven.Server
+```
+
+Registering as service in Windows:
+```
+.\rvn.exe windows-service register --service-name RavenDB4
+```
+
+Running as service in Linux, add to your daemon script:
+```
+<path/to/ravendb>/Server/Raven.Server --daemon
+```
+
+
+Hello World Example:
+--------------------
+1. Launch RavenDB server instance as follows:
+```
+   <path/to/ravendb>/Server/Raven.Server --ServerUrl=http://localhost:8080
+```
+2. Install dotnet core sdk. See : [Downloads](https://www.microsoft.com/net/download) and [PowerShell](https://github.com/PowerShell/PowerShell/releases)
+3. Open terminal and type:
+
+```
+   mkdir HelloWorld
+   cd HelloWorld
+   dotnet new console 
+```
+
+4. Replace the following files content:
```

Comment:
Nope.
Have them _manually_ create a new db and the sample data, then just add the snippets they need to read an employee